lib/pages/home/feedback_card.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
+import 'package:acs_upb_mobile/pages/class_feedback/service/feedback_provider.dart';
+import 'package:acs_upb_mobile/pages/class_feedback/view/classes_feedback_checklist.dart';
+import 'package:acs_upb_mobile/pages/classes/model/class.dart';
+import 'package:acs_upb_mobile/pages/classes/service/class_provider.dart';
+import 'package:acs_upb_mobile/widgets/info_card.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class FeedbackCard extends StatefulWidget {
```

Comment:
Maybe call it `FeedbackNudge`? I wouldn't call it notification since we don't have those implemented yet, but it's not really a card similar to the other ones we have.